src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
+const RENT_CAR_COST = 40;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const DISCOUNT_SHORT_TERM = 20;
+const DISCOUNT_LONG_TERM = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const RESULT = days * RENT_CAR_COST;
```

Comment:
Actually, `RESULT` does not have to be in capital letters. Think of it in a way that constants like above: `SHORT_TERM` `LONG_TERM` should never change in your code, like `Math.PI` or `Math.SQRT2`. But a variable with keyword `const` does not always have to be written in capital letters, this is a regular const that will change based on `days`, so it should be `const result`